dojo/tools/openvas/xml_parser.py

```diff
@@ -38,11 +41,16 @@ def get_findings(self, filename, test):
 
             finding = Finding(
                 title=str(title),
+                test=test,
```

Comment:
Why do we need this?

Reply:
when we try to use finding.save(), always appear `An exception error occurred during the report import: Finding has no test.`, seeing others parser, all of them, used this `test` and helped to this problem

Reply:
Curious as this would break importing data; but in any case, it might also be best submitted in a separate PR as a bugfix.